src/store/User/selectors.ts

```diff
@@ -0,0 +1,9 @@
+import type { RootStore } from "src/store/store";
+import type { State } from "./slice";
+
+export const getIsAuthValue = (store: RootStore): State["isAuth"] =>
```

Comment:
вообще любая переменная это какое-то value, поэтому этот постфикс не несет никакой дополнительной нагрузки, а лишь удлиняет название. Поэтому стараемся избегать в названиях "value" / "data" / 'item" и тд
```suggestion
export const getIsAuth = (store: RootStore): State["isAuth"] =>
```